### PR TITLE
Created autoClimber subsystem

### PR DESCRIPTION
**Pull request by boinary**
potentiometer input should be read here and tell the rollers on the climber to spinny spin

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/autoClimber.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ * Add your docs here.
+ */
+public class autoClimber extends Subsystem {
```
Comment (KenwoodFox) [inline src/main/java/frc/robot/subsystems/autoClimber.java]: I recomend a name change here.

**Comment by KenwoodFox** [comment]
You can continue adding commits or pull this code rn, and re-open the branch at your next commit.

**Comment by boinary** [comment]
We should sit down and talk through this as I'm not quite understanding. Thank you Joe

**Comment by boinary** [comment]
From my understanding, I should create `subsystems/Climber` and call a command `autoClimb()` inside of `subsystems/Climber` ?

**Comment by KenwoodFox** [comment]
Exactly correct yes!

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+public class Climber extends Subsystem {
+  // Put methods for controlling this subsystem
+  // here. Call these from Commands.
+
+  @Override
+  public void initDefaultCommand() {
+    // Set the default command for a subsystem here.
+    // setDefaultCommand(new MySpecialCommand());
+    
+    setDefaultCommand(new AutoClimb());
```
Comment (KenwoodFox) [inline src/main/java/frc/robot/subsystems/Climber.java]: I think we may have to do this differently, but no worries, for now we can leave autoclimb as the default.

**Pull request merged**